lib/Plugin.class.php

```diff
@@ -53,6 +53,7 @@ public function instantiateCompiler()
         $this->compiler = new WPLessCompiler;
         $this->compiler->setVariable('stylesheet_directory_uri', "'" . get_stylesheet_directory_uri() . "'");
         $this->compiler->setVariable('template_directory_uri', "'" . get_template_directory_uri() . "'");
+        if(!WP_DEBUG) $this->compiler->setFormatter('compressed');
```

Comment:
Could you follow the codebase style by wrapping the `if` statement block in curly braces?

Then I will merge the change :-)

Thanks!